src/Validation/src/Microsoft.Extensions.Validation.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should this package be named Microsoft.Extensions.Validation.Experimental? And always use a pre-release version even when we release 10?

Reply:
I don't think so since there are non-experimental types in this package that we'll keep using and won't be moved to the runtime. The ones that are going to be moved are marked as experimental on a per-type basis.